sdk/core/core-https/src/policies/exponentialRetryPolicy.ts

```diff
@@ -17,6 +17,13 @@ const DEFAULT_CLIENT_RETRY_COUNT = 10;
 const DEFAULT_CLIENT_RETRY_INTERVAL = 1000;
 const DEFAULT_CLIENT_MAX_RETRY_INTERVAL = 1000 * 64;
 
+const retryHttpStatusCodes: number[] = [
+  // Client error responses
+  408, // Request timeout
+  425, // Too early
```

Comment:
is there an Azure service that actually returns this?

Reply:
I don't know, I added those that seemed reasonable to add 😅

I only know about the 429.